Samples/PublicSamples/EchoBot/src/EchoBot/Controllers/JoinCallController.cs

```diff
@@ -0,0 +1,57 @@
+﻿using EchoBot.Bot;
+using EchoBot.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using System.Net;
+
+namespace EchoBot.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
```

Comment:
Assuming we add more endpoints to this controller in the future, do we want the base path to be joinCall? Or should we move the route to the specifc join call method?